mlflow/R/mlflow/.build-doc.R

```diff
@@ -4,6 +4,7 @@
 # (https://github.com/quantsch/Rd2md/tree/6b4825579a2df8a22898316d93729384f92a756b)
 # with a single extra commit to fix rendering of \link tags between methods in R documentation.
 devtools::install_git("https://github.com/smurching/Rd2md", ref = "mlflow-patches")
+devtools::install_version("roxygen2", "7.1.2")
```

Comment:
`roxygen2 7.1.2` is successfully installed:

https://app.circleci.com/pipelines/github/mlflow/mlflow/11885/workflows/1e83583b-2623-4ca0-b895-33db63015911/jobs/20087?invite=true#step-102-1716